README.md

```diff
@@ -26,9 +26,14 @@ The standard developer tools, including a C compiler, GNU Make, Bash, and Git. M
 # You'll run `make update` after each `git pull` in the future to keep those submodules updated.
 make wakunode2
 
+# Run with DNS bootstrapping
+./build/wakunode2 --dns-discovery --dns-discovery-url=DNS_BOOTSTRAP_NODE_URL
+
 # See available command line options
 ./build/wakunode2 --help
 ```
+You can use nodes specified [here](https://github.com/waku-org/nwaku/blob/master/apps/chat2/chat2.nim#L375-L378) as bootstrap nodes.
```

Comment:
Would it make more sense to point to the https://fleets.status.im/ for the DNS discovery infor?